src/components/dock/dock-button/dock-button.tsx

```diff
@@ -7,7 +7,7 @@ import { createRandomString } from '../../../util/random-string';
  */
 @Component({
     tag: 'limel-dock-button',
-    shadow: false,
+    shadow: true,
```

Comment:
It's good that this change is in a separate commit. But the other two changes in this commit should be in a `fixup! fix(dock-button): emit event when dock item is clicked for custom components` commit instead.

Reply:
This change also breaks styling in `dock-button.scss` where we use the selector `.dock-item:not(.selected) .button:not(.disabled)`.

Reply:
Ah, Victor's comment made me look at this a little closer, and if I recall correctly, I think I was involved in creating this component (not entirely sure, it was before my vacation), and I think we made it non-shadowed on purpose, precisely because we needed that styling to work, and the button is a "sub-component" of `limel-dock`, only intended for use inside the dock, and so it's still isolated from the rest of the consuming application by `limel-dock`'s shadow DOM.

@Befkadu1 Was there a specific reason for making this component shadowed here, or was it just an effort to align it with our usual conventions? If the latter, while definitely a laudable effort, I think we can keep this component non-shadowed.

Reply:
I thought we disabled the encapsulation by mistake. If it was made on purpose, I would revert back the change. I was encapsulation. I was trying to shield the component’s style.

Reply:
Cool! As I said, that's a good intention! 👍 ❤️ But yes, in this specific case, it was done on purpose 😊

Reply:
Ok then if there is no other comments, I am squashing the commits.